.env.tpl

```diff
@@ -0,0 +1,7 @@
+PORT=8000
```

Comment:
Can you use `.env` instead of `.env.tpl` ? No time to rename stuff, let's get developers to jump right into it

Reply:
When you run `./trust build` file .env will be created.

Reply:
What if I'm not running docker? by default, it should be there
If .env not working as expected this could be a different problem